cli/flox/src/utils/errors.rs

```diff
@@ -347,12 +348,12 @@ pub fn format_core_error(err: &CoreEnvironmentError) -> String {
 
         // catch package conflict error:
         // https://github.com/flox/flox/issues/857
-        CoreEnvironmentError::BuildEnv(CallPkgDbError::PkgDbError(PkgDbError {
-            exit_code: error_codes::BUILDENV_CONFLICT,
-            context_message: Some(ContextMsgError { message, .. }),
-            ..
-        })) => message.to_string(),
-        CoreEnvironmentError::BuildEnv(CallPkgDbError::PkgDbError(PkgDbError {
+        // CoreEnvironmentError::BuildEnv(CallPkgDbError::PkgDbError(PkgDbError {
```

Comment:
Can this be deleted?

Reply:
Yeah, i mentioned in another comment that we lost ability to catch these with buildenv.nix and have to rely on build logs to not be too nasty